build.html

```diff
@@ -12,7 +12,7 @@
         <p>Use the <strong>Refresh</strong> button below to try again.</p>
       </div>
       <div class="empty-area">
-        <h4>The are no new conversations</h4>
+        <h4>There are no new conversations</h4>
```

Comment:
@hcarneiro 

**There are no conversations.**

Don't think we need **new**. Or am I not understanding the scenario correctly?

_These are not the conversations you are looking for......._